### DISCUSSION TITLE
Disable reset password

### DISCUSSION THREAD

**Discussion by elliot-gh** [firefly-iii/firefly-iii#6748] [General]
I was wondering if a feature to specifically disable the password reset feature could be included for implementation in the future. I like having emails configured for automated transaction notifications, though since I own the database I would rather not have the password reset function work to prevent potential email spam (I'm aware that the password reset form does not confirm the existence of accounts).

**Comment by JC5** [comment]
It already has spam protection, you can't actually call it that often.
